# The input used in vehicle control
# Steering angle and acceleration
float64[] delta
float64[] acc